src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -589,4 +589,40 @@ describe('Test suite for mappings on existing concepts', () => {
     wrapper.find('button#removeAnswer').simulate('click');
     expect(props.removeAnswer).toHaveBeenCalledWith(props.selectedAnswers[0].frontEndUniqueKey);
   });
+  it('should call updateSourceEventListener function when editting a mapping to a new  source that is not `CIEL`', () => {
+    const event = {
+      target: {
+        tabIndex: 0,
+        name: 'source',
+        value: 'a234',
+      },
+    };
+    const url = '1234';
+    const instance = wrapper.find('EditConcept').instance();
+    instance.state.mappings[1] = {
+      map_type: 'NARROWER THAN',
+      source: 'Datatypes',
+    };
+    instance.updateSourceEventListener({ target: { tabIndex: 0, name: 'source', value: 'Datatypes' } });
+    instance.updateSourceEventListener(event, url);
+    expect(instance.state.mappings[1].source).toEqual('Datatypes');
+  });
+  it('should call updateSourceEventListener function when editing a mapping  to the `CIEL` source', () => {
```

Comment:
I do not seem to understand testing that updateSourceEventListener is called yet when you are explicitly calling it yourself in the test.

Reply:
I have adjusted this description accordingly